### PR TITLE
update list of tested OSes accordingly with metadata.json

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
rebased version of GH-402

**Comment by mmoll** [comment]
Debian/buster is already bringing NodeJS 10, so using 8.x as default will not work, as the package is not downgraded, leading to a test fail.

Looking at https://nodejs.org/en/about/releases/ I'd even lean towards setting the default to 12.x.

File: manifests/params.pp [hunk 1]
```diff
@@ -11,7 +11,7 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_release                = undef
-  $repo_url_suffix             = '8.x'
+  $repo_url_suffix             = '10.x'
```
Comment (bastelfreak) [inline manifests/params.pp]: is this change required? If so it qualifies as a backwards-incompatble change?

Comment (mmoll) [reply manifests/params.pp]: https://github.com/voxpupuli/puppet-nodejs/pull/414#issuecomment-575895848 :)

Comment (bastelfreak) [reply manifests/params.pp]: yeah, I'm fine with 12

Comment (mmoll) [reply manifests/params.pp]: There we go... :green_heart:

**Comment by juniorsysadmin** [comment]
Also consider updating the `repo_url_suffix` section of the README

**Comment by mmoll** [comment]
> Also consider updating the repo_url_suffix section of the README

Done.
